examples/task-plugins/dioptra_custom/pixel_threshold/pixelthreshold.py

```diff
@@ -124,11 +124,11 @@ def create_pt_dataset(
     target_size: Tuple[int, int] = image_size[:2]
     adv_data_dir = Path(adv_data_dir)
 
+    # attack requires clip values to be defined
+    keras_classifier.set_params(**{"clip_values": (0, 1)})
```

Comment:
Purely stylistic suggestion but this can be removed and replaced with setting `clip_values` as a parameter to `load_wrapped_tensorflow_keras_classifier`. Not sure if this is something we actually want to do though as including it here is potentially more fitting as it's related to the pixel threshold attack directly..